packages/fluent/scss/timedurationpicker/index.scss

```diff
@@ -0,0 +1,28 @@
+// Module meta
+$_kendo-module-meta: (
+    name: "timedurationpicker",
+    dependencies: (
+        "button",
+        "input",
+        "popup",
```

Comment:
add dependency to icon package. This should be done in the default and classic themes as well